process/tests/test_process.py

```diff
@@ -211,6 +222,14 @@ def test_check_filter_user(mock_process, reset_process_list_cache, aggregator, d
     aggregator.assert_metric('system.processes.number', value=2, tags=generate_expected_tags(instance))
 
 
+@patch('psutil.Process', return_value=MockProcess())
+def test_use_oneshot(mock_process, reset_process_list_cache, aggregator, dd_run_check):
```

Comment:
What is this meant to test? `MockProcess.oneshot` looks like a noop.

Reply:
I only really had to setup MockProcess.oneshot just to mock a context manager since the tests were failing without one. Looking at it now, this test is redundant since use_oneshot is true by default.